src/plugins/date-picker/index.tsx

```diff
@@ -307,9 +311,13 @@ const datePickerPlugin: MessagePluginFactory = ({ styled }) => {
       const flatpickrLocaleId = getFlatpickrLocaleId(localeId);
       let locale = l10n[flatpickrLocaleId];
       const enableTime = !!data.enableTime;
+
       const timeTemp = data.time_24hr ? 'H:i' : 'h:i'; //12-hour format without AM/PM
       const timeWithSeconds = data.enableSeconds ? `${timeTemp}:S` : timeTemp;
       const timeFormat = data.time_24hr ? timeWithSeconds :`${timeWithSeconds} K` //12-hour format with AM/PM
+
+      const dateFormatString = enableTime ? `${dateFormat} ${timeFormat}` : dateFormat;
+      const dateFormatLocalString = `L${enableTime ? " LT" : ""}${data.enableSeconds ? "S" : ""}`;
```

Comment:
We need to make sure to include the information about seconds-related change in changelog. 

I see the reason behind it, though we need to make it visible that the Webchat will now include seconds in the output if the setting enables them in the picker.

Please mention this in the PR description.

Reply:
Basically, we had a bug in this part. We included the seconds when user enabled it and added a custom date format.
In case of no custom, we are localizing the date without seconds.

I agree, we need to include this fix on the changelog.

Reply:
I reopen this thread since I found a specific case to handle. I will fix in the next commit

Reply:
fixed

Reply:
For the records @kwinto:
After depper investigation and discussion we agreed on keep this part of code without any new change.